src/js/components/Select/doc.js

```diff
@@ -161,3 +161,53 @@ export const doc = Select => {
 
   return DocumentedSelect;
 };
+
+export const themeDoc = {
+  'select.background': {
+    description: 'The background color used for Select.',
+    type: 'string',
+    defaultValue: undefined,
+  },
+  'select.container': {
+    description: 'The container of the Select component.',
+    type: 'string',
+    defaultValue: undefined,
+  },
+  'select.container.extend': {
+    description:
+      'Any additional style for the container of the Select component.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'select.control': {
```

Comment:
I don't think we need to document this intermediate key. `select.control.extend` should be sufficient.